### PR TITLE
COMMERCE-6890 SF Rename channelGroupId to commerceChannelGroupId

### PR DESCRIPTION
**Pull request by dsitu**
https://issues.liferay.com/browse/COMMERCE-6890

### CODE REVIEW

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-product-item-selector-web/src/main/java/com/liferay/commerce/product/item/selector/web/internal/CPDefinitionItemSelectorView.java [hunk 1]
```diff
@@ -83,11 +83,11 @@ public void renderHTML(
 		HttpServletRequest httpServletRequest =
 			(HttpServletRequest)servletRequest;
 
-		long channelGroupId = ParamUtil.getLong(
+		long commerceChannelGroupId = ParamUtil.getLong(
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-product-item-selector-web/src/main/java/com/liferay/commerce/product/item/selector/web/internal/CPDefinitionItemSelectorView.java:86]: Should the constant be renamed too?

Comment (dsitu) [reply modules/apps/commerce/commerce-product-item-selector-web/src/main/java/com/liferay/commerce/product/item/selector/web/internal/CPDefinitionItemSelectorView.java:86]: oh yeah, thanks for catching that!

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
ci:forward

**Pull request closed without merging**